frappe/public/js/frappe/chat.js

```diff
@@ -949,14 +949,14 @@ frappe.chat.pretty_datetime   = function (date)
 {
     const today    = moment()
     const instance = date.moment
-    
+        
     if ( today.isSame(instance, "d") )
```

Comment:
Shouldn't these utilities be in `datetime.js` ? They can be used elsewhere, why namespace them in chat?

Reply:
We already have a utility for pretty datetime called `comment_when`